### ISSUE TITLE
create an existing tmux issue raises an error

### ISSUE THREAD

**Issue by 0xIslamTaha** [threefoldtech/js-ng#111]
```
JS-NG> j.core.executors.tmux.create_session('Dream')                                                                                                                                                                 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/js-ng/jumpscale/core/executors/tmux.py", line 8, in create_session
    return server.new_session(session_name, kill_session=kill_if_exists)
           │                  │                          └ False
           │                  └ 'Dream'
           └ <libtmux.server.Server object at 0x7f29dcec6cc0>
  File "/root/.cache/pypoetry/virtualenvs/js-ng-py3.7/lib/python3.7/site-packages/libtmux/server.py", line 519, in new_session
    raise exc.TmuxSessionExists('Session named %s exists' % session_name)
          │                                                 └ 'Dream'
          └ <module 'libtmux.exc' from '/root/.cache/pypoetry/virtualenvs/js-ng-py3.7/lib/python3.7/site-packages/libtmux/exc.py'>
libtmux.exc.TmuxSessionExists: Session named Dream exists

Session named Dream exists
JS-NG> j.core.executors.tmux.create_session('Dream')                                                                                                                                                                 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/js-ng/jumpscale/core/executors/tmux.py", line 8, in create_session
    return server.new_session(session_name, kill_session=kill_if_exists)
           │                  │                          └ False
           │                  └ 'Dream'
           └ <libtmux.server.Server object at 0x7f29dcec6cc0>
  File "/root/.cache/pypoetry/virtualenvs/js-ng-py3.7/lib/python3.7/site-packages/libtmux/server.py", line 519, in new_session
    raise exc.TmuxSessionExists('Session named %s exists' % session_name)
          │                                                 └ 'Dream'
          └ <module 'libtmux.exc' from '/root/.cache/pypoetry/virtualenvs/js-ng-py3.7/lib/python3.7/site-packages/libtmux/exc.py'>
libtmux.exc.TmuxSessionExists: Session named Dream exists

Session named Dream exists

```

**Comment by AhmedHanafy725** [comment]
Solved in #420 
Use `create_session` with `kill_if_exists` to kill the session if it exists and create new one.
```python
j.core.executors.tmux.create_session('Dream', kill_if_exists=True)
```
And `get_session` will get the session if it exists and create a session if not.
```python
j.core.executors.tmux.get_session('Dream')
```

**Comment by Dina-Abd-Elrahman** [comment]
- verified.
